docs/vectorizer.md

```diff
@@ -140,13 +152,15 @@ The view includes all columns from the blog table, plus the following additional
 To find the closest embeddings to a query, use this canonical SQL query:
 
 ```sql
-SELECT 
-   chunk,
-   embedding <=> <query embedding> as distance
+SELECT chunk,
+   embedding <=> ai.openai_embed('text-embedding-3-small', 'my query string', _dimensions=>768)  as distance
```

Comment:
The `ai.openai_embed` function allows users to directly do semantic search in SQL, should we replace all occurences of `<query embedding>` with it?

Reply:
@jgpruitt what do we need to do to make this work on cloud @jgpruitt I am wondering about secrets.

Reply:
it works on cloud. The secret handling is awkward to deal with securely, but that's an issue across the board -- not specific to this. I think this is a good change.

Reply:
we need the secrets/env-var extension. some way to query cloud secrets

Reply:
you can manually do `select set_config('ai.openai_api_key', '<your-key-here>', false);` to set it in your db session, but that's no fun

Reply:
How do you use this function on cloud right now? Would you have to provide they api key as a parameter?